### PR TITLE
primary branch can be main

### PR DESCRIPTION
**Pull request by tkatzir**
## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://github.com/demisto/etc/issues/39616

### CODE REVIEW

File: demisto_sdk/commands/common/git_util.py [hunk 1]
```diff
@@ -422,21 +422,37 @@ def check_if_remote_exists(self, remote):
...
-
+            branch = ''
+            if prev_ver:
+                branch = prev_ver
+            else:
+                # Try to get the main branch
+                for current_remote in self.repo.remotes:
+                    for current_remote_ref in current_remote.refs:
+                        current_remote_ref_str = str(current_remote_ref)
+                        print('*** , ' + str(type(current_remote_ref)) + ', current_remote_ref: ' + str(current_remote_ref))
```
Comment (dorschw) [inline demisto_sdk/commands/common/git_util.py:445]: are these prints relevant in production?

**Review by dorschw — CHANGES_REQUESTED** [review]
@tkatzir Please run the nightly tests on this branch (see @MosheEichler's message in the [1.5.6 channel](https://panw-global.slack.com/archives/C02T5QC5LKX), or contact him or me for further help) with content's master, and update the status here

File: demisto_sdk/commands/common/git_util.py (hunk 1 above)
Comment (tkatzir) [reply demisto_sdk/commands/common/git_util.py]: Removed

File: demisto_sdk/commands/common/hook_validations/pack_unique_files.py [hunk 2]
```diff
@@ -64,9 +66,12 @@ class PackUniqueFilesValidator(BaseValidator):
     """PackUniqueFilesValidator is designed to validate the correctness of content pack's files structure.
     Existence and validity of this files is essential."""
 
+    git_util = GitUtil(repo=Content.git())
+    main_branch = git_util.handle_prev_ver()[1]
```
Comment (yaakovi) [inline demisto_sdk/commands/common/hook_validations/pack_unique_files.py:70]: same here

**Review by yaakovi — APPROVED** [review]
looks good to me

**Pull request merged**